samples/tab-navbar-menu/ts/README.md

```diff
@@ -0,0 +1,103 @@
+---
+page_type: sample
+description: Sample personal tab navbar-menu.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
```

Comment:
I couldn't see linking this sample with main root readme file.

Reply:
I have updated the changes.